example/quick-start/base.yaml

```
@@ -7,23 +7,13 @@ metadata:
-          value: koupleless_base
-        - name: KUPLELESS_ARKLET_MQTT_PASSWORD
-          value: public
-        - name: KUPLELESS_ARKLET_MQTT_CLIENT_PREFIX
-          value: koupleless
-        - name: KUPLELESS_ARKLET_CUSTOM_TUNNEL_CLASSNAME
-          value: com.alipay.sofa.koupleless.arklet.tunnel.mqtt.MqttTunnel
-        - name: KUPLELESS_ARKLET_CUSTOM_BASE_METADATA_CLASSNAME
-          value: com.alipay.sofa.web.base.metadata.MetadataHook
\ No newline at end of file
+        - name: MODULE_CONTROLLER_ADDRESS
+          value: {YOUR_MODULE_CONTROLLER_IP}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)